src/core/lib/iomgr/work_serializer.h

```diff
@@ -39,19 +39,22 @@ namespace grpc_core {
 // All callbacks scheduled on a WorkSerializer instance will be executed
 // serially in a borrowed thread. The API provides a FIFO guarantee to the
 // execution of callbacks scheduled on the thread.
-// When a thread calls Run() with a callback, the thread is considered borrowed.
-// The callback might run inline, or it might run asynchronously in a different
-// thread that is already inside of Run(). If the callback runs directly inline,
-// other callbacks from other threads might also be executed before Run()
-// returns. Since an arbitrary set of callbacks might be executed when Run() is
-// called, generally no locks should be held while calling Run().
+// When a thread calls Schedule() with a callback, the callback is added to the
```

Comment:
I think the original text here was a bit more informative, since we still expect `Run()` to be the most common method used here.